macros/src/main/scala/packaged/insidePackageObject/package.scala

```diff
@@ -2,7 +2,7 @@ package packaged
 
 import scala.gestalt._
 
-package object macros2 {
+package object insidePackageObject {
   inline def plus(a: Int, b: Int): Int = meta {
     q"$a + $b"
   }
```

Comment:
BTW, it seems the directory structure is unnecessary for package objects, it's possible to have them just defined in the same directory in any Scala file. If it's true, maybe we can clean up the directory a little bit to put all definitions inside on folder?